bindings/python/pinocchio/visualize/panda3d_visualizer.py

```diff
@@ -0,0 +1,129 @@
+import multiprocessing
+import numpy as np
+import os
+import sys
+import uuid
+import warnings
+
+from pinocchio import libpinocchio_pywrap as pin
```

Comment:
You can replace this with
```
from .. import libpinocchio_pywrap as pin
```

Reply:
Hi @gabrielebndn , thanks for the comments. The PEP8 standart says [Absolute imports are recommended, as they are usually more readable and tend to be better behaved...](https://www.python.org/dev/peps/pep-0008/#imports). Do you have a special motivation of using relative imports in this case?

Reply:
Using the absolute import here may cause weird conflicts if there is more than one version of Pinocchio installed, or if you are doing `make test` (which works on a local copy in the `build` folder, not on the installed version). Therefore, in Pinocchio's Python code we are using explicit relative imports.

PEP8 also says [However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose](https://www.python.org/dev/peps/pep-0008/#imports). I would say this is our case.

I really think relative imports are better in this context, but maybe @nim65s can give us more insight

Reply:
Ok, switched to relative imports

Reply:
fixed